test/unit/process/test_troe_rate_constant.cpp

```diff
@@ -85,5 +85,5 @@ TEST(TroeRateConstant, AnalyticalTroeExampleBC)
   double k1 =
       k_0 * 42.2 / (1.0 + k_0 * 42.2 / k_inf) * pow(0.9, 1.0 / (1.0 + (1.0 / 0.8) * pow(log10(k_0 * 42.2 / k_inf), 2)));
 
-  EXPECT_EQ(k, k1);
+  EXPECT_NEAR(k, k1, 1e-8);
```

Comment:
I did not understand why we need to use `expect_near` rather than `expect_eq`. I saw some constants were declared as integer for `troe_params` (e.g., .k0_B_), but a real number was used for the analytical solution (e.g., 167.0 for k_0 calculation). Could this be a problem?

Reply:
According to Kyle, this issue only happens when using the `DEBUG` mode. I checked the relative difference here and it is smaller than 1e-16, which to me could be safely ignored.